### DISCUSSION TITLE
ARRIRAWToolbox?

### DISCUSSION THREAD

**Discussion by terrylb** [latenitefilms/BRAWToolbox#171] [Feature Requests]
Any plans to develop an ARRIRAWToolbox?

**Comment by latenitefilms** [comment]
It's theoretically possible - but I'd imagine the user-base would be pretty tiny? Could you explain more why you want direct ARRIRAW support?

**Comment by terrylb** [comment]
Hi Chris,

Thank you for the quick reply. We received some ARRIRAW footage and being an FCP shop, I couldn't find a way to work with it directly. Ended up using Arri's ART app to export ProRes 4444 files. Apparently it's supported in DVR and PP. I was wondering if you were considering an FCP extension like BRAWToolbox.

Thanks,
-Terry
